templates/home.html

```diff
@@ -38,29 +103,68 @@ <h4 class="white-color header-padding">Welcome, {% firstof user.get_full_name us
+                    <img class="img-phone" src="{% static "img/perdiem-invest.png" %}">
+                </div>
+                <div id="rightcolumn">
+                    <h3 class="title">Buy and sell shares in music</h3>
+                    <p>
+                        <strong>We make it easy for anyone to invest in music.</strong> Never worry about contracts or legal fees. Artists can raise money by selling shares in their music projects. This gives people an incentive to fund the artists music as they can earn a return on their investment based on amount of sales.
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div id="how-page" class="about">
+                <div id="column-wrapper">
+                    <div id="leftcolumn">
+                        <h3 class="title">Track your earnings</h3>
+                        <p>
+                            <strong>We take care of everything.</strong> Investors can track all of their earnings from all of their artists in one place. Never worry about distribution, payments, and registration. All you have to do invest, promote your artists, and watch your money grow.
+                        </p>
+                        <p>
+
+                        </p>
+                    </div>
+                    <div id="rightcolumn">
+                        <img class="img-phone" src="{% static "img/perdiem-earnings.png" %}">
+                    </div>
+            </div>
+        </div>
+        <div id="stats-page" class="about">
+            <img class="img-phone" src="{% static "img/perdiem-all-devices.png" %}">
+            <h3 class="title">Be your own record label</h3>
+            <p>
+                <strong>The best way to invest in music.</strong> Record Labels take a lot of work. They require huge teams of people all dedicated to the success of an album. With PerDiem, Artists can raise money from multiple labels and investors, allowing many teams all pushing for their success.
+            </p>
+            <h5><a href="{% url 'resources' %}">Learn More</a></h5>
+        </div>
+        <div id="invest-page" class="about">
+            <div class="break">
+                <hr>
+            </div>
+            <div id="column-wrapper">
+                <h3 class="title">Free for artists</h3>
+                <h5 class="">Distribution, Funding, Payment Splits, and everything else you'll ever need</h5>
+                <div class="center extra-padding">
+                    <a class="button sign-up uppercase" href="{% url 'artist_application' %}">Apply Now</a>
                 </div>
             </div>
-            <div class="discover extra-padding">
-                <h5 class="white-color">PerDiem is a new way for artists fund their music.</h5>
-                <h5 class="white-color">Artists can raise money by selling shares in their music projects.</h5>
+            <div>
+                <h3 class="title">$1 / Trade</h3>
```

Comment:
Ideally we shouldn't have this $1 hardcoded since it can be different from one campaign to another, right?

Reply:
Ahh.. I meant that the cost to trade was only $1. Maybe I should re-word this section